lib/common/monitoring/alarms/UsageAlarmFactory.ts

```diff
@@ -75,6 +82,31 @@ export class UsageAlarmFactory {
     });
   }
 
+  /**
+   * Allows addition of multiple metrics from the same class.
+   * @param percentMetric 
+   * @param usageType Will be added to Alarm name
+   * @param props 
+   * @param disambiguator 
+   * @returns 
```

Comment:
These don't really add value as-is. Either leave them out or fill out the descriptions.